plugins/chat/app/models/chat/message.rb

```diff
@@ -266,7 +266,7 @@ def upsert_mentions
     end
 
     def in_thread?
-      self.thread_id.present?
+      self.thread_id.present? && self.chat_channel&.threading_enabled
```

Comment:
Is there a case where a chat message does not have a chat channel? 🤔

Reply:
Good question 🧐 

I don't _think_ there's one. Removed the `&` and re-ran the tests and it's all 🟢.